package.json

```diff
@@ -1,22 +1,25 @@
 {
   "name": "tsheets-sdk",
-  "version": "0.1.3",
+  "version": "1.0.0-0",
   "description": "TSheets JS SDK for full API access.",
   "main": "./lib/index.js",
+  "module": "./es/tsheets-sdk.js",
   "files": [
     "src",
     "lib"
   ],
   "scripts": {
     "lint": "eslint .",
-    "lint:fix": "npm run lint -- --fix",
-    "clean": "rimraf lib coverage",
-    "test": "mocha -R spec --compilers js:babel-core/register ./test/setup.js ./test/**/*.spec.js",
-    "test:cov": "babel-node $(npm bin)/isparta cover --report lcov ./node_modules/mocha/bin/_mocha -- ./test --recursive",
+    "lint:fix": "yarn lint -- --fix",
+    "clean": "rimraf lib coverage es",
+    "test": "nf run -- mocha --compilers js:babel-core/register ./test/setup.js",
+    "test:all": "yarn test ./test/**/*.spec.js",
+    "test:cov": "babel-node $(npm bin)/isparta cover --report lcov $(npm bin)/_mocha -- ./test --recursive",
     "build:lib": "babel src --out-dir lib",
-    "build": "npm run clean && npm run build:lib",
-    "watch": "npm run build:lib -- --watch",
-    "prepublish": "npm run clean && npm run lint:fix && npm run test && npm run build"
+    "build": "yarn clean && yarn build:lib",
```

Comment:
Can we leave these npm scripts as just `npm *` since then they will work with either?